content/docs/command-reference/add.md

```diff
@@ -169,6 +169,12 @@ not.
   [remote storage](/doc/command-reference/remote) to transfer external target to
   (can only be used with `--to-remote`).
 
+- `-j <number>`, `--jobs <number>` - Only used if `--to-remote` is also passed.
+  Parallelism level for DVC to upload data to remote storage. The default value
+  is 4 \* cpu_count(). For SSH remotes, the default is 4. Note that the default
+  value can be set using the jobs config option with dvc remote modify. Using
+  more jobs may speed up the operation.
```

Comment:
For closer consistency with other refs that have this option:
```suggestion
- `-j <number>`, `--jobs <number>` -
  parallelism level for DVC to transfer data when using `--to-remote`. The default value
  is `4 \* cpu_count()`. For SSH remotes, the default is `4`. Using
  more jobs may speed up the operation.
```

Reply:
Committed. I removed "Note that the default value can be set using the jobs config option with dvc remote modify" too. It's a good note though, but it kind of overcomplicates the already-complicated description. I wonder if there's a better place to comment on this... ⌛

Reply:
> removed "Note that the default value can be set using the jobs config option...

Maybe we should change the text (in all refs) to something like "The default is 4*cpu (can be set with `dvc remote modify jobs`) :...

> Linked to https://dvc.org/doc/command-reference/remote/modify#available-parameters-for-all-remotes